src/hooks/bxt.rs

```diff
@@ -141,6 +141,7 @@ pub struct OnTasPlaybackFrameData {
     pub strafe_cycle_frame_count: u32,
     pub prev_predicted_trace_fractions: [f32; 4],
     pub prev_predicted_trace_normal_zs: [f32; 4],
+    pub rendered_viewangles: [f32; 3],
```

Comment:
Does this need a _v2 function name and a change on bxt side?

Reply:
Yea we need to change the `on_tas_playback_frame()` function to match. https://github.com/YaLTeR/bxt-rs/pull/101 also makes changes to the playback frame data so this PR (a lot shorter) needs merging first to avoid weird conflicts.